### PR TITLE
Update to 0.5.0 

### PR DESCRIPTION
**Pull request by Hongzhi-liu**
Updated Chinese version number to 0.5.0 to match VDK versions

### CODE REVIEW

**Comment by OsiJr** [comment]
@Hongzhi-liu, I'm not sure why every line has been changed? Did line endings change or some other misconfiguration?

**Comment by Hongzhi-liu** [comment]
@OsiJr 
I refer to the line by line translation of "vaultclient/builds/assets/lang/enAU.json".
I don't know why all the lines have changed.
And it has been tested and is correct when used.

File: builds/assets/lang/zhCN.json [hunk 1]
```diff
@@ -1,514 +1,573 @@
...
-  "viewShedVisibleColour": "可见面颜色",
-  "viewShedHiddenColour": "背面颜色"
-}
+{
+  "_LanguageInfo": {
+    "langkey": "zh",
+    "countrykey": "CN",
+    "localname": "简体中文",
+    "englishname": "Simplified Chinese",
+    "translationby": "深圳优立视觉科技有限公司",
```
Comment (samsinsane) [inline builds/assets/lang/zhCN.json]: There's a tab at the end of this line that shouldn't be there.

**Comment by samsinsane** [comment]
@Hongzhi-liu I assume you made this change on a Windows machine? It's likely due to a misconfiguration of the line endings settings, I suggest you checkout [this link](https://stackoverflow.com/questions/10418975/how-to-change-line-ending-settings), you want to have your config set to checkout Windows style and checkin Unix style. As this branch has two commits, it's best that you recreate the commit (copy the file, checkout latest master, paste the file over the top) and then create another PR - in the future, try to avoid using the "Update branch" in GitHub, it provides little value when you do translations.

**Comment by OsiJr** [comment]
I've fixed the line endings and merged this manually.

**Pull request closed without merging**